strongbox-commons/src/main/java/org/carlspring/strongbox/artifact/coordinates/NugetArtifactCoordinates.java

```diff
@@ -0,0 +1,51 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import org.carlspring.maven.commons.util.ArtifactUtils;
+
+/**
+ * @author Sergey Bespalov
+ *
+ */
+public abstract class NugetArtifactCoordinates extends AbstractArtifactCoordinates
+{
+
+    private static final String ID = "id";
+    private static final String VERSION = "version";
+
+    public NugetArtifactCoordinates()
+    {
+        defineCoordinates(ID, VERSION);
+    }
+
+    @Override
+    public String getId()
+    {
+        return getCoordinate(ID);
+    }
+
+    @Override
+    public void setId(
+                      String id)
+    {
+        setCoordinate(ID, id);
+    }
+
+    @Override
+    public String getVersion()
+    {
+        return getCoordinate(VERSION);
+    }
+
+    @Override
+    public void setVersion(
```

Comment:
@sbespalov , 

This is just a cosmetic remark: could please format first parameter of the methods to be on the same line as the method and then align all other parameters, under the first character?

For example:

    public void setBlaBla(String version,
                          String foo)

Could fix this across the rest of the code as well, please? Thanks! :)